Command/AnonymizeDataCommand.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the ekino/data-protection-bundle project.
+ *
+ * (c) Ekino
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Ekino\DataProtectionBundle\Command;
+
+use Ekino\DataProtectionBundle\Meta\AnonymizedMetadataBuilder;
+use Ekino\DataProtectionBundle\Meta\AnonymizedMetadataValidator;
+use Ekino\DataProtectionBundle\QueryBuilder\AnonymizedQueryBuilder;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class AnonymizeDataCommand
+ *
+ * @author Benoit Mazière <benoit.maziere@ekino.com>
+ */
+final class AnonymizeDataCommand extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected static $defaultName = 'ekino-data-protection:anonymize';
+
+    /**
+     * @var AnonymizedMetadataBuilder
+     */
+    protected $anonymizedMetadataBuilder;
+
+    /**
+     * @var AnonymizedMetadataValidator
+     */
+    protected $anonymizedMetadataValidator;
+
+    /**
+     * @var AnonymizedQueryBuilder
+     */
+    protected $anonymizedQueryBuilder;
+
+    public function __construct(
+        AnonymizedMetadataBuilder $anonymizedMetadataBuilder,
+        AnonymizedMetadataValidator $anonymizedMetadataValidator,
+        AnonymizedQueryBuilder $anonymizedQueryBuilder
+    )
+    {
+        parent::__construct();
+
+        $this->anonymizedMetadataBuilder   = $anonymizedMetadataBuilder;
+        $this->anonymizedMetadataValidator = $anonymizedMetadataValidator;
+        $this->anonymizedQueryBuilder      = $anonymizedQueryBuilder;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setDescription('Anonymize database based on entities annotations')
+            ->addOption('force', null, InputOption::VALUE_NONE, 'Set this parameter to execute this action')
```

Comment:
`dry-run` more than `force` no ?

Reply:
(and reverse utilisation of course)

Reply:
as it is an irreversible transformation/truncate of data, i prefer this way (as DropDatabaseDoctrineCommand) and let the user add the force option to actually run the transformation/truncate of data